main/ertp/api/amount-math.md

```diff
@@ -21,8 +21,8 @@ specified when `issuerKit()` creates the issuer and brand.
 
 We recommend you import the two `AssetKind` values from `@agoric/ERTP` instead of making the 
 strings yourself. 
-- `AssetKind.NAT` (`nat`): Used with fungible assets. `amount` `values` are natural numbers (non-negative `BigInts`).
-- `AssetKind.SET` (`set`): Used with non-fungible assets. `amount` `values` are objects or records with multiple properties.
+- `AssetKind.NAT` (`nat`): Used with fungible assets. Valuess are natural numbers using the JavaScript  [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) type to avoid overflow risks from using the usual JavaScript `Number` type.
```

Comment:
```suggestion
- `AssetKind.NAT` (`nat`): Used with fungible assets. Values are natural numbers using the JavaScript  [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) type to avoid overflow risks from using the usual JavaScript `Number` type.
```